ruby_on_rails/rails_basics/deployment.md

```diff
@@ -1,5 +1,5 @@
 ### Introduction
-Before continuing our web development journey, we must address one important task - showcasing our hard work to the world.
+Before continuing our web development journey, we must address one important task—showcasing our hard work to the world.
```

Comment:
Are you sure this is correct? these don't seem like two words that can be hyphenated together.

Reply:
My change switched it from a hyphen (-) to an em dash (—). Using an em dash in this context serves to define the final clause (i.e., "showcasing …") as a summarizing/expanding statement. See [Merriam-Webster](https://www.merriam-webster.com/words-at-play/em-dash-en-dash-how-to-use) for more details.

If the em dash feels wrong to you, another option would be to use a colon:
"… we must address one important task: showcasing our hard work …"

If you don't like either of those options, I think your original form was perfectly readable as it was. Grammar rules help provide consistency, but readability is all that really matters.

Reply:
ah sorry, my mistake. I really like your colon suggestion. Can we go with that please?